cpp/src/decisiontree/decisiontree.cu

```diff
@@ -114,18 +115,17 @@ void decisionTreeClassifierFit(const raft::handle_t &handle,
                                const int ncols, const int nrows, int *labels,
                                unsigned int *rowids, const int n_sampled_rows,
                                int unique_labels,
-                               DecisionTree::DecisionTreeParams tree_params,
```

Comment:
These 4 decisionTreeFit functions are effectively the same except for types. Is there a nice way to condense them? 

Same for predict.

Reply:
Actually, unless I am mistaken, these functions never gets used in cuml? So we could hypothetically remove this entire file.

Reply:
yep, will remove them shortly rory. 👍🏾